server/views/partials/layout.html

```diff
@@ -12,7 +12,7 @@
       window.dataLayer = window.dataLayer || [];
       function gtag(){dataLayer.push(arguments);}
       gtag('js', new Date());
-      gtag('config', '{{ googleAnalyticsKey }}');
+      gtag('config', '{{ googleAnalyticsKey }}', {{ { title: '' } if googleAnalyticsOverrideTitle else {} }});
```

Comment:
Will this switch all analytics off for when override is true?

Reply:
No, it should just override the title value sent to GA.